bears/vcs/git/GitCommitBear.py

```diff
@@ -241,7 +241,7 @@ def check_body(self, body,
                                'HEAD commit. Please add one.')
             return
 
-        if body_regex and not re.search(body_regex, body):
+        if body_regex and not re.fullmatch(body_regex, body.strip()):
```

Comment:
There should only be a single trailing `\n`.  Please only remove that, instead of `strip()`, which will mean junky git messages can slip through.

Otherwise the users [git config](https://git-scm.com/docs/git-commit#git-commit---cleanupltmodegt) needs fixing, and it should be an error.